ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -80,9 +80,16 @@ - (void)acquireToken:(NSString *)eventName
         
         [event setCorrelationId:self.correlationId];
         [event setClientId:_requestParams.clientId];
+        [event setAuthority:_context.authority];
         [event setExtendedExpiresOnSetting:[_requestParams extendedLifetime]? AD_TELEMETRY_YES:AD_TELEMETRY_NO];
         [event setPromptBehavior:_promptBehavior];
-        [event setUserInformation:[[result tokenCacheItem] userInformation]];
+        if ([result tokenCacheItem])
+        {
+            [event setUserInformation:[[result tokenCacheItem] userInformation]];
```

Comment:
Please use consistent dot syntax. As a rule-of-thumb if it's a property use dot syntax, so this should be `result.tokenCacheItem.UserInformation`